### DISCUSSION TITLE
Modifying SDL stereotype color

### DISCUSSION THREAD

**Discussion by System0X** [plantuml/plantuml#1546] [Q&A]
Hi i want to modify the color of SDL objects eg:
```plantuml
@startuml
#green:SDL Shape;
#pink:input<
#orange:output>
@enduml
```
But for the entire diagram something like:

```plantuml
@startuml
skinparam ActivityBackgroundColor green
skinparam Activity<<input>>BackgroundColor pink
skinparam Activity<<output>>BackgroundColor orange

:SDL Shape;
:input<
:output>
@enduml
```

**Comment by The-Lum** [comment]
Hello @System0X, and all,

For that you can use `style` _(global style: for the entire diagram)_ as:
```plantuml
@startuml
<style>
activity {
  BackgroundColor green
}
.input {
  BackgroundColor pink
}
.output {
  BackgroundColor orange
}
</style>

:SDL Shape;
:input; <<input>>
:output; <<output>>
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/TOv12eCm44NtSuevGTt74AJjiZjF47H8W-aan8aWfNStQfVjxlDvlF-xMMoMyWgWPra36R23-CNBYcz0lDfXSZaM7cynn8mk4p5yuE8v5Vc99C_JJiGY_v2OBJkga6w-ew3w-mFxfqtKWZhaBMfz16D0dRAzEbFjEkAn_ju0)](https://www.planttext.com/?text=TOv12eCm44NtSuevGTt74AJjiZjF47H8W-aan8aWfNStQfVjxlDvlF-xMMoMyWgWPra36R23-CNBYcz0lDfXSZaM7cynn8mk4p5yuE8v5Vc99C_JJiGY_v2OBJkga6w-ew3w-mFxfqtKWZhaBMfz16D0dRAzEbFjEkAn_ju0)

Ref.:
- #1423

Know issue:
- #1527
- #1538

_Enjoy,_
Regards,
Th.
